R/comp_corr.R

```diff
@@ -13,8 +13,17 @@
 #' (1 highly correlated, 0 no correlation)
 #'
 #' @examples
-#' comp.corr(array(rnorm(40), c(5, 8)), time = t(array(c(0, 5, 10, 15, 20, 25,
-#' 30, 35), c(8, 5))), C = 10)
+#' ##upon preparing the data using the best.lag ouput, however the best.lag is sampled here
+#' lagged <- prep.data(array(rnorm(30), c(3, 10)), timepoints = seq(0, 45, 5),
+#'           lags = sample(c(0, 1, -1, 2, -2), size = 3))
+#' comp.corr(data = lagged$data, time = lagged$time, C = 10)
+#'
+#' ## example two with actual best.lag output
+#' data2 <- array(rnorm(120), c(10, 12))
```

Comment:
Should we call this `randdata` or `dataset` or something else to help distinguish `data2` from `data`?